src/Compilers/CSharp/Portable/Symbols/LocalSymbol.cs

```diff
@@ -321,9 +321,9 @@ public object ConstantValue
 
         internal abstract ImmutableArray<Diagnostic> GetConstantValueDiagnostics(BoundExpression boundInitValue);
 
-        public bool IsRef => RefKind == RefKind.Ref;
+        public bool IsRef => RefKind != RefKind.None;
```

Comment:
Consider updating some compiler tests for this.

Reply:
I added new tests along with symbol display.